src/canvas/tileLayer.js

```diff
@@ -0,0 +1,87 @@
+geo.canvas.tileLayer = function () {
```

Comment:
Okay, let's talk about it at some point. I will look at this code in more detail.